app/models/rapids/work_process.rb

```diff
@@ -1,11 +1,37 @@
 module RAPIDS
   class WorkProcess
-    def self.initialize_from_response(work_process_data)
-      ::WorkProcess.new(
-        title: work_process_data["title"],
-        minimum_hours: work_process_data["minHours"],
-        maximum_hours: work_process_data["maxHours"]
-      )
+    class << self
+      def initialize_from_response(work_process_data)
+        tasks = work_process_data["tasks"]
+
+        work_process = ::WorkProcess.new(
+          title: work_process_data["title"],
+          minimum_hours: work_process_data["minHours"],
+          maximum_hours: work_process_data["maxHours"]
+        )
+
+        if competencies_available?(tasks)
+          competencies = process_competencies(tasks)
+          work_process.competencies = competencies
+        end
+
+        work_process
+      end
+
+      private
+
+      def competencies_available?(tasks)
+        tasks&.first&.present?
+      end
+
+      def process_competencies(tasks)
+        tasks_titles = tasks.first.split("; ")
```

Comment:
Can we split on optional whitespace in case the whitespace is sometimes missing? `split(/\s+;\s+/)`

Reply:
@littleforest thanks! Added a test for this